resources/dicts/thoughts/alive/kitten.json

```diff
@@ -129,32 +175,18 @@
         "thoughts": [
             "Thinks they saw a StarClan cat in their dreams",
             "Is mesmerized by the night sky",
-            "Thinks they're the leader of their own Clan",
+            "Thinks theyre the leader of their own clan",
```

Comment:
i pointed this out last time, but it still hasn't been fixed?

Reply:
I'll try to fix that.